content/en/docs/distributions/azure/deploy/install-kubeflow.md

```diff
@@ -10,178 +10,132 @@ deploy Kubeflow on Azure.
 ## Prerequisites
 
 - Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux)
-- Install and configure the [Azure Command Line Interface (Az)](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
-  - Log in with ```az login```
+- Install the [Azure Command Line Interface](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
 - Install Docker
-  - For Windows and WSL: [Guide](https://docs.docker.com/docker-for-windows/wsl/)
+  - For Windows and Windows Subsystem for Linux (WSL): [Guide](https://docs.docker.com/docker-for-windows/wsl/)
   - For other OS: [Docker Desktop](https://docs.docker.com/docker-hub/)
```

Comment:
Actually don't need docker for this deployment. Comes later when adding the containers to the registry in the guide: https://www.kubeflow.org/docs/distributions/azure/azureendtoend/